keyboards/massdrop/alt/keymaps/favorable-mutation/config.h

```diff
@@ -0,0 +1,137 @@
+#define SR_EXP_RCLK_PIN             14
+/* Shift Register Output Enable configuration (MCU to ShiftRegister.OE_N) */
+#define SR_EXP_OE_N_PORT            PB
+#define SR_EXP_OE_N_PIN             15
+/* SERCOM port to use for Shift Register SPI */
+/* DATAOUT and SCLK must be configured to use hardware pins of this port */
+#define SR_EXP_SERCOM               SERCOM2
+/* Shift Register SPI Data Out configuration (MCU.SERCOMx.PAD[0] to ShiftRegister.SER) */
+#define SR_EXP_DATAOUT_PORT         PA
+#define SR_EXP_DATAOUT_PIN          12
+#define SR_EXP_DATAOUT_MUX          2
+/* Shift Register SPI Serial Clock configuration (MCU.SERCOMx.PAD[1] to ShiftRegister.SRCLK) */
+#define SR_EXP_SCLK_PORT            PA
+#define SR_EXP_SCLK_PIN             13
+#define SR_EXP_SCLK_MUX             2
+
+/* Debug LED (Small LED Located near MCU) */
+#define DEBUG_LED_ENABLE            1
+#define DEBUG_LED_PORT              PA
+#define DEBUG_LED_PIN               27
+
+/* Additional debugging ports */
+/* PCB M21 */
+#define DEBUG_PORT1_ENABLE          1
+#define DEBUG_PORT1_PORT            PB
+#define DEBUG_PORT1_PIN             3
+/* PCB M23 */
+#define DEBUG_PORT2_ENABLE          1
+#define DEBUG_PORT2_PORT            PB
+#define DEBUG_PORT2_PIN             17
+/* PCB M25 */
+#define DEBUG_PORT3_ENABLE          1
+#define DEBUG_PORT3_PORT            PA
+#define DEBUG_PORT3_PIN             20
+
+/* Debug Boot Tracing - During boot sequence, ground this pin to halt and display debug code using Debug LED */
+/* This is useful in determining which hardware device may have malfunctioned or is improperly configured */
+/* Feature is automatically disabled after successful boot */
+/* PCB M27 */
+#define DEBUG_BOOT_TRACING_ENABLE   1
+#define DEBUG_BOOT_TRACING_PORT     PB
+#define DEBUG_BOOT_TRACING_PIN      23
+
+/* Debounce reduces chatter (unintended double-presses) - set 0 if debouncing is not needed */
+#define DEBOUNCE 5
+
+/* Mechanical locking support. Use KC_LCAP, KC_LNUM or KC_LSCR instead in keymap */
+//#define LOCKING_SUPPORT_ENABLE
+/* Locking resynchronize hack */
+//#define LOCKING_RESYNC_ENABLE
+
+/* Force boot in NKRO mode */
+//#define FORCE_NKRO
+
+/*
+ * Feature disable options
+ *  These options are also useful to firmware size reduction.
+ */
+
+/* disable debug print */
+//#define NO_DEBUG
+
+/* disable print */
+//#define NO_PRINT
+
+/* disable action features */
+//#define NO_ACTION_LAYER
+//#define NO_ACTION_TAPPING
+//#define NO_ACTION_ONESHOT
+//#define NO_ACTION_MACRO
+//#define NO_ACTION_FUNCTION
+
+#define RGB_MATRIX_KEYPRESSES
+#define RGB_MATRIX_LED_PROCESS_LIMIT 15
+#define RGB_MATRIX_LED_FLUSH_LIMIT 10
+
+// toggle one-shot layer keys after this many presses
+#define ONESHOT_TAP_TOGGLE 3
+
+// set time held before Mod-Tap key sends mod code
+#define TAPPING_TERM 1000
+
+// permissive hold: if holding down a Mod-Tap key while pressing another, it
+// acts as a mod, even within the tapping term
+#define PERMISSIVE_HOLD
```

Comment:
Most of this file is identical to the keyboard level config.h, and should be removed. All you need here are these lines, and `#pragma once` at the top.

Reply:
Did not realize it would inherit like that! Removed; thanks so much for reviewing!